docs/book/src/cronjob-tutorial/pprof-tutorial.md

```diff
@@ -0,0 +1,61 @@
+# Using Pprof for Controller Profiling
+
+Pprof is a Go based library designed for profiling the binary in order to gather CPU, and Memory related data. Pprof is helpful in finding any sort of bottlenecks that are appearing either at Memory, CPU, or both levels.
+
+Pprof itself runs as part of the the Controller. You can extract the statistics, and visualize them on an interactive website. Furthermore, **you don't need to install this package seperately**. Pprof comes packaged as part of Kubebuilder `controller-runtime` [dependency](https://github.com/kubernetes-sigs/controller-runtime/pull/1943) within the scafolded project.
+
+For further information about the Pprof usage, check out the official [docs](https://github.com/google/pprof).
+
+# How to use Pprof in your Codebase
```

Comment:
It is a sub-section so need to be `##` and not `#`, either I think something more like seems fit better, wdyt

```suggestion
## Enabling Pprof 

To enable `pprof` for your manager, follow these steps:
```

Reply:
We do not use Codebase term usually, that does not seems to fit
Need to be something like  Enabling Pprof